test.js

```diff
@@ -41,12 +41,18 @@ describe('Plugin', () => {
       dict: [{key: '__KEY__', value: '__VALUE__'}],
     }}});
 
-    it('should replace key with value', () => {
+    it('should not replace when key is absent', () => {
```

Comment:
I think the test was correct but the description was copied and pasted from elsewhere.  I updated the description to match the test.